File: TypeScript/Cryptography/get-sha3-512-of-string.ts
import * as crypto from "crypto"

function main(): void {
    // Get the sha3-512 of a string
    console.log(getSHA3512OfString("Hello World!"))
}

main()

// Get the sha3-512 of a string
function getSHA3512OfString(givenValue: string): string {
    return crypto.createHash("sha3-512").update(givenValue).digest("hex")
}

File: Python/Cryptography/get_sha_3_512_of_string.py
import hashlib


# Get the sha-3-512 of the string.
def get_sha_3_512_of_string(provided_string):
    return hashlib.sha3_512(provided_string.encode()).hexdigest()


def main():
    # Get the sha-3-512 of the string.
    print(get_sha_3_512_of_string("Hello World!"))


main()
